src/vulkan/gen/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ add_executable(shaders-gen ${SOURCES})
 if(WIN32)
     target_link_libraries(shaders-gen "$ENV{VULKAN_SDK}/Lib/vulkan-1.lib" "$ENV{VULKAN_SDK}/Lib/shaderc_combined.lib")
 elseif(APPLE)
+    # we should check for M1 vs Intel here too
+    add_compile_definitions(SPACEMESH_VULKAN_COMPATIBILITY_NEEDED=1)
```

Comment:
ah yeah it works only when it's in src/CmakeLists.txt so @poszu you're right. Forgot to remove that.

@fasmat it will not work on `gen` level. I can think how to add new list to `vulkan` level though.

Reply:
I see, if it doesn't work here, then I would also be OK with having it in the root `CMakeLists.txt`